### ISSUE TITLE
Can I use others' vhd imageto deploy a VM?

### ISSUE THREAD

**Issue by wangjj1989** [Azure/azure-quickstart-templates#2825]
Hi all,
I create a vm vhd image and set the access type of the container of the image to 'container'(means others can read at least). Is it possible for others(not share resource group to he/she) to use that vhd image to deploy a VM? For example, use the Uri of the vhd image in the template file and deploy a VM.

**Comment by MCKLMT** [comment]
Hi,
Currently, the image should be in the same storage account than the disk of the VM which will be deployed. If you want to share your image with others, you should give the rights on the image file.

**Comment by ostclilideng** [comment]
@wangjj1989 yes, you can please refer to below section, bold one is the link for your vhd image.
                "storageProfile": 
                {
                    "osDisk" : 
                    {
                        **"image": 
                        {
                            "uri": "[concat('http://',variables('StorageAccountName'),'.blob.core.windows.net/vhds/','based.vhd')]"
                        },**
                        "osType": "Linux",
                        "name": "ICA-RG-ExtensionTest-lili72-1-4-9-47-42-role-0-OSDisk",
                        "vhd": 
                        {
                            "uri": "[concat('http://',variables('StorageAccountName'),'.blob.core.windows.net/vhds/','osdisk.vhd')]"
                        },
                        "caching": "ReadWrite",
                        "createOption": "FromImage"
                    }
                }
